Stripe/STPPaymentCardTextFieldViewModel.m

```diff
@@ -12,6 +12,7 @@
 #define FAUXPAS_IGNORED_IN_METHOD(...)
 
 @implementation STPPaymentCardTextFieldViewModel
+@dynamic valid;
```

Comment:
same here – i don't think this @dynamic declaration is necessary, since we implement the getter